### PR TITLE
fix(login): inception, new header.php for iframe

### PR DESCRIPTION
**Pull request by victorvassilev**
not to allow iFrame to get to login.php page

### CODE REVIEW

File: www/include/core/header/header.iframe.php [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+require_once $classdir . '/centreonFeature.class.php';
+require_once SMARTY_DIR . "Smarty.class.php";
+
+/*
+ * Create DB Connection
+ *  - centreon
+ *  - centstorage
+ */
+$pearDB = new CentreonDB();
+$pearDBO = new CentreonDB("centstorage");
```
Comment (sc979) [inline www/include/core/header/header.iframe.php]:
```suggestion
$pearDBO = $dependencyInjector['realtime_db'];
```

File: www/include/core/header/header.iframe.php [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+$time_start = microtime_float();
+
+$advanced_search = 0;
+
+/*
+ * Include
+ */
+include_once(realpath(dirname(__FILE__) . "/../../../../bootstrap.php"));
+
+require_once "$classdir/centreonDB.class.php";
```
Comment (sc979) [inline www/include/core/header/header.iframe.php]: when using dependencyInjector, you don't need this class anymore

Comment (sc979) [inline www/include/core/header/header.iframe.php]: ! -> $classdir may not be defined
I found only one affectation in the "centreon.config.php.template" . But I may be wrong. Why don't you use a  $path = __DIR__ . "XXX" and use it instead ?

Comment (sc979) [inline www/include/core/header/header.iframe.php]: PSR2 -> "$classdir/centreonDB.class.php" -> $classdir . "/centreonDB.php" like line 63

**Comment by kduret** [comment]
I close This PR cause it's not mergeable anymore.
And I think we've may some fixes about login page redirection.
We'll reopen this if problem reappears

**Pull request closed without merging**